.github/workflows/main.yml

```diff
@@ -18,7 +18,7 @@ on:
 
 jobs:
   appimage:
```

Comment:
I think you should have a test job here first, and have appimage depend on it. You don't want to build new releases if they don't pass tests after merging.

Reply:
Where? It's still not showing a "needs" line for me. Maybe you forgot to push?